### PR TITLE
add support of PIE for r2

### PR DESCRIPTION
**Pull request by matrix1001**
Sorry for this super late PR. This is just a small problem due to PIE. I am not very familiar to the api of `pwndbg`. But this change just fixed the PIE problem of r2.

### CODE REVIEW

File: pwndbg/commands/radare2.py [hunk 1]
```diff
@@ -25,8 +25,11 @@ def r2(arguments, no_seek=False):
 
     # Build up the command line to run
     cmd = ['radare2', filename]
+    addr = pwndbg.regs.pc
+    if pwndbg.wrappers.checksec.pie_status() == "PIE enabled":
```
Comment (disconnect3d) [inline pwndbg/commands/radare2.py]:
```suggestion
    if pwndbg.elf.is_pie():
```

Can you check if that works fine?

Comment (matrix1001) [reply pwndbg/commands/radare2.py]: Exception occured: r2: 'module' object has no attribute 'is_pie' (<type 'exceptions.AttributeError'>)

elf is a module. 

Maybe we need an object?

Comment (disconnect3d) [reply pwndbg/commands/radare2.py]: Oh it should be:
```python
pwndbg.elf.get_elf_info(filepath).is_pie
```

Comment (matrix1001) [reply pwndbg/commands/radare2.py]: Checked.  But that line should be:
```
if pwndbg.elf.get_elf_info(filename).is_pie:
```

Comment (disconnect3d) [reply pwndbg/commands/radare2.py]: Sure, can you change it to this? :)

Comment (matrix1001) [reply pwndbg/commands/radare2.py]: Can you see it ?

**Review by disconnect3d — APPROVED** [review]

File: pwndbg/commands/radare2.py (hunk 1 above)
Comment (disconnect3d) [reply pwndbg/commands/radare2.py]: Yeah, thx!

**Pull request merged**